blocks/providers/providers.css

```diff
@@ -0,0 +1,100 @@
+.providers{
+    padding: 0 10px;
+}
+.providers__heading{
+    margin: 40px 10px;
```

Comment:
remove margin on the top here and setting the spacing as a padding on the block level seems like more secure way to align elements in a block. Providers__heading overrides the margins set in the heading class. Maybe just remove this class and use block level padding in lieu of a modifier.